### ISSUE TITLE
Is it possible to use search-source with a few collections at once?

### ISSUE THREAD

**Issue by chompomonim** [meteorhacks/search-source#23]
I have to implement global search in my app. It should show search results from Clients, Projects, Orders and Suppliers collections into one list.

Is search-source good choice to do that? If yes, any examples? If not, then also ok, I will just look for some other solutions... ;)

Thanks for any answers.

**Comment by arunoda** [comment]
Yes. You can search for multiple collections and need to generate a single
array of results.

**Comment by chompomonim** [comment]
@arunoda you mean I should do something like:

    GlobalSearch = new SearchSource(['clients', 'projects'], fields, options);

**Comment by btbjosh** [comment]
+1  I'm really looking for an answer to this question as I need a full app solution for searching like this.

**Comment by derwaldgeist** [comment]
+1

**Comment by insaneinc01** [comment]
+1 Searching multiple collections - some guidance would be really great.

Another question: how do we handle **multiple** keyword search in one collection? For example, 
I need to search for 
`manager sales london contract`
through an `Applicants` collection which has various fields like location, skill, role, type etc. 

What I could do right now is make multiple searches for each keyword, but merging the returned array for all keywords and then filtering/sorting is turning cumbersome. 
Any ideas/advice please?
